R/register.R

```diff
@@ -23,16 +23,29 @@ register_main_language <- function(main_language = "en", project_path = ".") {
   }
 
   config_lines <- brio::read_lines(config_path)
-  config_lines <- c(
-    config_lines,
-    "",
-    "babelquarto:",
+  mainlanguage_config <- c(
     "  languagecodes:",
     sprintf("  - name: %s", main_language),
     sprintf('    text: "Version in %s"', main_language),
     sprintf("  mainlanguage: '%s'", main_language),
     sprintf("lang: %s", main_language)
   )
+  babelquarto_config_exists <- which(grepl("babelquarto:", config_lines))
```

Comment:
```suggestion
  babelquarto_config_exists <- grep("babelquarto:", config_lines)
```

I do this "mistake" so often myself that it took me a while to notice it!

Reply:
I'm afraid my base R is not up to spotting that… I just copied what was done elsewhere 🙈
I'll try to remember that one!

Reply:
Apparently the dev version of lintr has a linter for that: https://lintr.r-lib.org/dev/reference/which_grepl_linter.html 

This made me open #63